### PR TITLE
[SV] Support for scenes

### PR DESCRIPTION
**Pull request by danielholm**
Adding support for scenes in Swedish.

### CODE REVIEW

**Comment by danielholm** [comment]
It's currently a conflict between light and scene while trying
$ pytest tests --language sv -k scene_HassTurnOn
E                               AssertionError: Expected light, got scene for slot domain for: slå på pluggtid i kök
E                               assert 'scene' == 'light'
E                                 
E                                 - light
E                                 + scene

tests/test_language_sentences.py:217: AssertionError
=================================================================================== short test summary info ===================================================================================
FAILED tests/test_language_sentences.py::test_scene_HassTurnOn[sv] - AssertionError: Expected light, got scene for slot domain for: slå på pluggtid i kök
========================================================================= 1 failed, 1 passed, 101 deselected in 0.63s =======================================================================

File: sentences/sv/_common.yaml [hunk 1]
```diff
@@ -353,6 +353,7 @@ expansion_rules:
   temperature: "{temperature}[°| grader] [{temperature_unit}]"
   färga: "(färga | ändra | byt)"
   volym: "{volume:volume_level}[[ ]%| procent]"
+  här: "(här|härinne|i detta rum|i rummet|i rummet jag är|i rummet här)"
```
Comment (danielholm) [inline sentences/sv/_common.yaml]:
```suggestion

```

**Review by danielholm — COMMENTED** [review]
Yes, that's the changes I want

**Comment by danielholm** [comment]
Should work now. Doing a build test.

**Comment by danielholm** [comment]
It's been a decade since i codes (used Bazar, I believe it was called), so you'll have to excuse my poor git skills. I do however believe that I re-added the README (rebase?) and removed my previous changes in that file. Should be good now - alas I have said so plenty a times now.
Cheer!!

**Review by larsdunemark — APPROVED** [review]

**Pull request merged**